packages/create-redwood-app/README.md

```diff
@@ -85,19 +85,25 @@ The installation script is built with [Yargs](https://github.com/yargs/yargs)
 The project codebase in `template/` uses [Yarn Workspace v1](https://classic.yarnpkg.com/en/docs/workspaces/) for a monorepo project containing the API and Web Sides. Redwood packages are included in `template/package.json`, `template/web/package.json`, and `template/api/package.json`, respectively.
 
 ### How to run create-redwood-app from your local repo and create a project
-Make sure you to first run `yarn install` in your project root.
+First, run `yarn install` in the root of the mono repo. Then, run `yarn install` again, inside the `packages/create-redwood-app` directory.
 
-Step into the `create-redwood-app` package and run the script:
+Then, we need to build the script. Step into the `create-redwood-app` package and run the script:
 
 ```bash
 cd packages/create-redwood-app
-yarn babel-node src/create-redwood-app.js /path/to/new/redwood-app
+yarn build
 ```
 
-This will create a new project using the local `template/` codebase
+_Note:_ You could use `yarn build:watch` instead of `yarn build` to watch for changes and rebuild automatically.
 
-> Note: the new project will install with the most recent stable Redwood package version by default
+This will generate `create-redwood-app.js` file inside the `dist` directory.
 
+To use the script, run `node` on that file (dist/create-redwood-app.js) and pass in the path to the new project:
+```bash
+node dist/create-redwood-app.js /path/to/new/redwood-app
+```
+
+> Note: the new project will install with the most recent stable Redwood package version by default
```

Comment:
It won't be the most recent stable version but the most recent major (i.e. `5.0.0` as opposed to `5.3.2`); just a consequence of our branch strategy. We release all minors/patches off a "next" branch, majors off main